### PR TITLE
style: ACL rules has no max-width

### PR DESCRIPTION
**Pull request by CamilleLegeron**
#674

There were a glitch on the ACL formula Editor, so add style to hide this glitch.
Screenshot from 2024-01-09 11-59-40

### CODE REVIEW

File: app/client/aclui/ACLFormulaEditor.ts [hunk 1]
```diff
@@ -133,6 +133,9 @@ const cssConditionInputAce = styled('div', `
   & .ace-chrome, & .ace-dracula {
     background-color: ${theme.accessRulesFormulaEditorBg};
   }
+  & .ace-chrome .ace_print-margin, & .ace-dracula .ace_print-margin {
+    width: 0px;
+  }
```
Comment (georgegevoian) [inline app/client/aclui/ACLFormulaEditor.ts:138]:
```suggestion
  &:not(:focus-within) .ace_print-margin {
    width: 0px;
  }
```

This hides the indicator line when the editor isn't focused. Feels like a good compromise. Wdyt?

**Review by georgegevoian — COMMENTED** [review]
Thanks @CamilleLegeron.

Just one suggestion.

File: app/client/aclui/ACLFormulaEditor.ts (hunk 1 above)
Comment (CamilleLegeron) [reply app/client/aclui/ACLFormulaEditor.ts:138]: Well, if you prefer it's good to me, I just does not understand why this separator line in the middle of the editor it's important. It's look like a visual glitch to me.

Comment (CamilleLegeron) [reply app/client/aclui/ACLFormulaEditor.ts:138]: Or it have to be closer to the indicators

Comment (CamilleLegeron) [reply app/client/aclui/ACLFormulaEditor.ts:138]: `right: 5px !important;
    left: unset !important;`

Comment (CamilleLegeron) [reply app/client/aclui/ACLFormulaEditor.ts:138]:
```suggestion
  & .ace-chrome .ace_print-margin, & .ace-dracula .ace_print-margin {
    right: 5px !important; 
    left: unset !important;
  }
```

Comment (georgegevoian) [reply app/client/aclui/ACLFormulaEditor.ts:138]: It's probably a good idea to keep it where it is while the editor has focus. It's consistent with the formula editor elsewhere in Grist, and it's common in Python code to limit lines to ~80 characters, which is where the line starts.

**Review by georgegevoian — APPROVED** [review]
Thanks @CamilleLegeron!

**Pull request merged**